org-formation/100-shared-dns/_tasks.yaml

```diff
@@ -0,0 +1,38 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+AppZone:  # Zone for application front-end DNS names, e.g. 'dca.app.sagebionetworks.org'
```

Comment:
I would be more specific since you might add other instances of application zones.  How about `SageBionetworksAppZone` or `AppSageBionetworksOrgZone`?

Reply:
done